spec/factories/addresses.rb

```diff
@@ -2,16 +2,32 @@
 
 FactoryBot.define do
   factory :address do
-    street { '1901 Main St' }
-    street2 { nil }
-    city { 'Vancouver' }
-    state { 'WA' }
-    postal_code { '98660' }
-    county { 'Clark' }
-
     latitude { nil }
     longitude { nil }
+    skip_api_validation { true } # dont hit the API in the test suite
+
+    trait :fake do
+      street { Faker::Address.street_address }
+      city { Faker::Address.city }
+      county { Faker::Address.street_name }
+      state { Faker::Address.state_abbr }
+      postal_code { Faker::Address.zip }
+    end
+
+    trait :wa do
+      street { "1901 Main St" }
```

Comment:
think if you were to run `rails cleanup:rubocop`, it'd change all these double quotes to single quotes